github/pulls_reviewers.go

```diff
@@ -13,18 +13,22 @@ import (
 // RequestReviewers creates a review request for the provided GitHub users for the specified pull request.
 //
 // GitHub API docs: https://developer.github.com/v3/pulls/review_requests/#create-a-review-request
-func (s *PullRequestsService) RequestReviewers(ctx context.Context, owner, repo string, number int, logins []string) (*PullRequest, *Response, error) {
+func (s *PullRequestsService) RequestReviewers(ctx context.Context, owner, repo string, number int, logins []string, teams []string) (*PullRequest, *Response, error) {
```

Comment:
@gmlewis, what do you think about this API decision?

An alternative to consider (that I can think of) is to create a `ReviewersRequest` struct which contains the two string slices, and have the user pass that in.

I'm pretty torn between the two, it seems they tradeoffs are pretty much equivalent and I'm not strongly in favor of either of them.